src/helpers/errorManager.js

```diff
@@ -0,0 +1,30 @@
+const AVAILABLE_CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function errorManager(originalText, targetCase) {
+  const errors = [];
+
+  if (!originalText) {
+    errors.push({
+      message: 'Text to convert is required. '
+        + 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
Better to move all messages as constant to separate file.
Also, this part of message is repeatable in 2 messages, for this you can create separate constant